deployments/kubernetes/install/helm/istio-cni/templates/istio-cni.yaml

```
@@ -132,6 +131,34 @@ spec:
               name: cni-bin-dir
             - mountPath: /host/etc/cni/net.d
               name: cni-net-dir
+      # This container installs the Istio CNI binaries
+      # and CNI network config file on each node.
+        - name: repair-cni
+          image: {{ .Values.repair.hub }}/istio-cni-repair:{{ .Values.repair.tag }}
```

Include the binary in the existing image and use it here:
```suggestion
          image: {{ .Values.hub }}/install-cni:{{ .Values.tag }}
```